docs/install-upgrade.md

```diff
@@ -22,6 +23,7 @@ Before you install the MongoDB Community Kubernetes Operator, you must:
    git clone https://github.com/mongodb/mongodb-kubernetes-operator.git
    ```
 4. Review the possible Operator [deployment scopes](#understand-mongodb-community-operator-deployment-scopes) and configure the Operator to watch other namespaces, if necessary.
+5. Configure the [MongoDB Docker image or container registry](#configure-the-mongodb-docker-image-or-container-registry), if necessary.
```

Comment:
Nit: Should we put the condition first? "If necessary, configure..." 
I know this messes up the parallelism of the list items a little bit. Up to you.

Reply:
removed conditional statement. Made last two steps optional instead.

Reply:
Nice, I like that better too.